app/views/estimates/_limits_table.html.slim

```diff
@@ -2,12 +2,17 @@ p.govuk-body = t(".calculation_information")
 p.govuk-body = t(".regulations_html")
 h2.govuk-heading-m = t(".limits_heading")
 p.govuk-body = t(".outgoings_explanation")
+- lower_limits = @model.level_of_help == "certificated" && !@estimate.upper_tribunal?
+- upper_limit_could_be_waived = @model.level_of_help == "certificated" && !@estimate.upper_tribunal?
 - t(".list").each do |reason|
```

Comment:
these both do the same thing, is having them both just for clarity as to why/why not a section would be displayed?